secp256k1-sys/src/lib.rs

```diff
@@ -530,7 +749,7 @@ extern "C" {
 //  In:      flags: which parts of the context to initialize.
 #[no_mangle]
 #[cfg(all(feature = "std", not(rust_secp_no_symbol_renaming)))]
-pub unsafe extern "C" fn rustsecp256k1_v0_4_0_context_create(flags: c_uint) -> *mut Context {
```

Comment:
nit: what is upstream's reason for versioning the API this way? Pretty minor, but if trying to stay close to upstream, maybe keeping this convention is a good idea?